dll/cpl/intl/lang/ro-RO.rc

```diff
@@ -39,83 +39,83 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Limbi de introducere și servicii de text", -1, 5, 5, 234, 52
-    LTEXT "Pentru a vedea/modifica limbile de introducere și serviciile de text, apăsați Detalii", -1, 12, 15, 220, 18
+    GROUPBOX "Servicii de text şi limbi pentru tastatură", -1, 5, 5, 234, 52
+    LTEXT "Pentru a vizualiza sau înlocui limbile şi metodele utilizate pentru introducerea de text, faceţi clic pe Detalii.", -1, 12, 15, 220, 18
     PUSHBUTTON "&Detalii…", IDC_DETAIL_BUTTON, 177, 34, 54, 14
-    GROUPBOX "Compatibilitate lingvistică suplimentară", -1, 5, 62, 234, 82
-    LTEXT "Cele mai multe limbi sunt instalate implicit. Pentru a instala mai multe limbi, selectați căsuța de selecție apropiată de mai jos.", -1, 12, 72, 220, 25
-    CHECKBOX "Instalează fișierele pentru script-uri comple&xe și pentru limbile cu scriere inversă", IDC_INST_FILES_FOR_RTOL_LANG, 12, 100, 215, 22, BS_MULTILINE | BS_TOP
-    CHECKBOX "Instalează fișierele pentru limbile din &estul Asiei", IDC_INST_FILES_FOR_ASIAN, 12, 119, 180, 22, BS_MULTILINE | BS_TOP
+    GROUPBOX "Suport pentru limbi suplimentare", -1, 5, 62, 234, 82
+    LTEXT "Majoritatea limbilor sunt instalate implicit. Pentru a instala limbi suplimentare, selectaţi caseta corespunzătoare.", -1, 12, 72, 220, 25
+    CHECKBOX "I&nstalare fişiere pentru script-uri complexe şi pentru limbi cu scriere de la dreapta la stânga (inclusiv Thai)", IDC_INST_FILES_FOR_RTOL_LANG, 12, 100, 215, 22, BS_MULTILINE | BS_TOP
+    CHECKBOX "In&stalare fişiere pentru limbile est-asiatice", IDC_INST_FILES_FOR_ASIAN, 12, 119, 180, 22, BS_MULTILINE | BS_TOP
 END
 
 IDD_ADVANCEDPAGE DIALOGEX 0, 0, 246, 230
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Avansate"
+CAPTION "Complex"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Limba pentru programe ne-Unicode", -1, 5, 5, 234, 90
+    GROUPBOX "Limbi pentru programele non-Unicode", -1, 5, 5, 234, 90
     COMBOBOX IDC_LANGUAGE_COMBO, 14, 75, 217, 160, CBS_DROPDOWNLIST | CBS_NOINTEGRALHEIGHT | CBS_AUTOHSCROLL | WS_VSCROLL | WS_TABSTOP | CBS_SORT
-    LTEXT "Această opțiune specifică limba de afișare pentru programele care nu utilizează Unicode în textele din meniuri și ferestrele de dialog. Opțiunea nu afectează aplicațiile Unicode, dar va afecta toți utilizatorii din sistem.", -1, 14, 18, 223, 33
-    LTEXT "Alegeți limba ce corespunde cu limba aplicațiilor ne-Unicode pe care doriți să le utilizați:", -1, 14, 55, 223, 18
-    GROUPBOX "Tabele de conversie cod-pagină", -1, 5, 101, 234, 88
+    LTEXT "Setările acestui sistem permit programelor non-Unicode să afişeze meniuri şi dialoguri în limba lor de origine. Acest fapt nu afectează programele Unicode, dar se aplică pentru toţi utilizatorii acestui computer.", -1, 14, 18, 223, 33
+    LTEXT "&Selectaţi o limbă care să corespundă versiunii de limbă a programelor non-Unicode pe care le veţi utiliza:", -1, 14, 55, 223, 18
+    GROUPBOX "Tabele de &conversie a paginilor de cod", -1, 5, 101, 234, 88
```

Comment:
which sense does it make to add an accelerator to a group-box?

Reply:
This is what MS did in XP/2k3. Here is the proof.

![2024-08-17 00_43_38-](https://github.com/user-attachments/assets/00d4f6fe-addc-4e2b-8f3c-bb01e7a7c57e)